src/components/scenes/Staking/StakeModifyScene.tsx

```diff
@@ -407,7 +407,9 @@ export const StakeModifyScene = (props: Props) => {
         }
         {
           // Render claim amount tile
-          modification === 'claim' || modification === 'unstake' ? stakePolicy.rewardAssets.map(asset => renderEditableQuoteAmountRow('claim', asset)) : null
+          !stakePolicy.rewardsNotClaimable && (modification === 'claim' || modification === 'unstake')
```

Comment:
This double-negative is pretty trippy. We could flip the whole thing around, `stakePolicy.rewardsNotClaimable || (modification !== 'claim' && modification !== 'unstake') ? null : stakePolicy.rewardAssets.map(asset => renderEditableQuoteAmountRow('claim', asset))`

Reply:
Took me a minute but yeah your replacement makes way more sense